todo_app/app.py

```diff
@@ -1,32 +1,44 @@
-from flask import Flask
-from flask import render_template, request, redirect, url_for
+from flask import Flask, render_template, redirect, url_for, request
 
+from todo_app.data import trello_items as trello
```

Comment:
This import assumes that the `trello_items` file is in the `data` folder, but it's currently one level higher - so either we should move it (my preferred solution) or we should update this to be correct, e.g.:
```
from todo_app import trello_items as trello
```